src/System.Resources.Analyzers/Core/MarkAssembliesWithNeutralResourcesLanguage.cs

```diff
@@ -48,9 +48,10 @@ public override void Initialize(AnalysisContext analysisContext)
             // this analyzer is safe from running concurrently.
```

Comment:
Not related to this change: This comment implies that the analyzer will never run concurrently. Something like "This analyzer can safely run concurrently." would be more accurate.